pkgs/tools/admin/salt/pepper/default.nix

```diff
@@ -1,17 +1,20 @@
 { lib
-, fetchurl
 , python2Packages
+, salt
 }:
 
 python2Packages.buildPythonApplication rec {
-  name = "salt-pepper-${version}";
-  version = "0.5.5";
-  src = fetchurl {
-    url = "https://github.com/saltstack/pepper/releases/download/${version}/${name}.tar.gz";
-    sha256 = "1wj1k64ly6af6qsmiizlx32jxh23a37smd9wb57l5zl0x8sfqq1n";
+  pname = "salt-pepper";
+  version = "0.7.5";
+  src = python2Packages.fetchPypi {
```

Comment:
I imagine this supports Python 3 since Salt does too - can we swap this to `pythonPackages`?